service-api/pom.xml

```diff
@@ -170,6 +174,20 @@
             <artifactId>feign-hystrix</artifactId>
             <version>${feign-core.version}</version>
         </dependency>
+
+        <!-- https://mvnrepository.com/artifact/org.testcontainers/junit-jupiter -->
+        <dependency>
+            <groupId>org.testcontainers</groupId>
+            <artifactId>junit-jupiter</artifactId>
+            <version>${test.containers.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-test</artifactId>
+            <version>2.6.2</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
do we need these dependencies? 
Please check and remove them if they are unused

Reply:
i added 2 dependencies in my last commit, i used them in AddNewTariffDtoTest class